demisto_sdk/commands/lint/tests/command_builder_test.py

```diff
@@ -7,7 +7,7 @@
 values = [[Path("file1.py")], [Path("file1.py"), Path("file2.py")]]
 
 
-@pytest.mark.parametrize(argnames="py_num , expected_exec", argvalues=[(3.7, 'python3'), (2.7, 'python')])
+@pytest.mark.parametrize(argnames="py_num , expected_exec", argvalues=[('3.7', 'python3'), ('2.7', 'python')])
```

Comment:
please add tests involving `3.10` and `3.10.1`

Reply:
done